extra/cockpit/foreman-cockpit.conf

```diff
@@ -0,0 +1,11 @@
+# Configuration directory for cockpit-ws
+XDG_CONFIG_DIRS=/etc/foreman/
+
+# Configuration file for foreman-session
+FOREMAN_COCKPIT_SETTINGS=/etc/foreman/cockpit/foreman-cockpit-session.yml
+
+# Address to listen on for cockpit-ws
+FOREMAN_COCKPIT_ADDRESS=127.0.0.1
+
+# Port to listen on for cockpit-ws
+FOREMAN_COCKPIT_PORT=9999
```

Comment:
And this needs to be Puppet-managed at some point.

Reply:
I'd like to avoid this and prefer systemd drop in files. We have started to use this in the installer more and more. The main benefit is that it avoids differences between Debian and RH systems where one has env files in `/etc/default` and the other in `/etc/sysconfig`. It is also easy to see the env with `systemctl cat $service`.

Reply:
Agreed, rebased.